core/storage/question/gae_models.py

```diff
@@ -48,6 +48,50 @@ def get_deletion_policy():
         return base_models.DELETION_POLICY.NOT_APPLICABLE
 
 
+class QuestionCommitLogEntryModel(base_models.BaseCommitLogEntryModel):
+    """Log of commits to questions.
+
+    A new instance of this model is created and saved every time a commit to
+    QuestionModel occurs.
+
+    The id for this model is of the form 'question-[question_id]-[version]'.
+    """
+
+    # The id of the question being edited.
+    question_id = datastore_services.StringProperty(indexed=True, required=True)
+
+    @staticmethod
+    def get_model_association_to_user():
+        """This model is only stored for archive purposes. The commit log of
+        entities is not related to personal user data.
+        """
+        return base_models.MODEL_ASSOCIATION_TO_USER.NOT_CORRESPONDING_TO_USER
+
+    @classmethod
+    def get_export_policy(cls):
+        """Model doesn't contain any data directly corresponding to a user.
+        This model is only stored for archive purposes. The commit log of
+        entities is not related to personal user data.
```

Comment:
Ditto here & above (per docstring changes I suggested for collections). Ditto elsewhere in PR.

Reply:
Done.